src/browser/bounds_changed_state_tracker.js

```diff
@@ -30,6 +30,16 @@ import { toSafeInt } from '../common/safe_int';
 
 const isWin32 = process.platform === 'win32';
 
+const DisableWindowGroupTracking = 'disable-window-group-tracking';
+
+function shouldTrack(action) {
```

Comment:
Could you write a gist in a comment here briefly explaining what this feature is?